src/soroban/api.ts

```diff
@@ -166,6 +166,7 @@ export namespace Api {
     ledgerClosedAt: string;
     pagingToken: string;
     inSuccessfulContractCall: boolean;
+    txHash: string;
```

Comment:
should this be denoted as optional field just in case any backwards compat possibilities with older rpc's that don't emit this yet?

Reply:
it should be fine since because this is just a schema definition and not parsed/enforced anywhere, it will actually just be undefined on the JS side anyway